bindings/python/src/OpenSpaceToolkitIOPy/IP/TCP/HTTP/Client.cpp

```diff
@@ -12,7 +12,10 @@ inline void OpenSpaceToolkitIOPy_IP_TCP_HTTP_Client(pybind11::module& aModule)
 
         .def_static("send", &Client::Send)
         .def_static("get", &Client::Get)
-        .def_static("fetch", &Client::Fetch)
+        .def_static(
+            "fetch", &Client::Fetch, pybind11::arg("url"), pybind11::arg("directory"), pybind11::arg("follow_count") = 0
```

Comment:
This is the correct naming convention I believe, but would love a second opinion